docs/conf.py

```diff
@@ -243,4 +243,4 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
```

Comment:
Ugh, I've just tweaked that file to add additional references in #52.  Are we really sure we want to standardize `docs/conf.py` across repositories?

Reply:
Or maybe you added that change manually (I don't see a template for it in the `meta/config/c-code` tree, anyway) to silence the warning?  In which case I will just resolve the conflict.